app/views/article/_index_default.html.erb

```diff
@@ -22,7 +22,14 @@
 <% if document.access_panels.online? && !document.eds_restricted? %>
   <ul class='document-metadata dl-horizontal dl-invert'>
     <% document.access_panels.online.links.each do |link| %>
-      <li><%= link_to(link.text, link.href) %></li>
+      <li>
+        <% if link.text =~ /^Find it in print/ -%>
```

Comment:
Okay, that one looks good, want to swap the `[WIP]` (or just remove it from the other one I suppose).